meson.build

```diff
@@ -33,6 +33,7 @@ wayland_protos = dependency('wayland-protocols')
 wlroots = dependency('wlroots', fallback: ['wlroots', 'wlroots'])
 gtkmm = dependency('gtkmm-3.0')
 dbusmenu_gtk = dependency('dbusmenu-gtk3-0.4', required: get_option('dbusmenu-gtk'))
+giounix = dependency('gio-unix-2.0', required: get_option('dbusmenu-gtk'))
```

Comment:
(Random note: I think this was supposed to have been always required. I'm building with it enabled though, not sure.)